docs/examples/sample/README.md

```diff
@@ -0,0 +1,23 @@
+# LeaderWorkerSet Features
+LeaderWorkerSet supports the following features.
+
+## Multi template for leader/worker pods
+
+Different templates for leaders and workers can be specified as seen [here](lws-multi-template.yaml),
+Leader template is an optional field. If not specified, it will default to the worker template.
+
+## Restart Policy
+
+There exists an API field called RestartPolicy to define the failure handling schematics for the pod group.
+On default, only failed pods will be restarted. When it is set to RecreateGroupOnRestart, it will recreate 
```

Comment:
```suggestion
By default, only failed pods will be automatically restarted. When the RestartPolicy is set to RecreateGroupOnRestart, it will recreate 
```